src/pages/Detailpage/Detailpage.js

```diff
@@ -1,13 +1,395 @@
-import React, { Component } from "react";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import Slider from "react-slick";
+import styled from "styled-components";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
 
-class Detailpage extends Component {
-  render() {
-    return (
-      <div>
-        <h1>hi</h1>
-      </div>
-    );
-  }
-}
+const Detailpage = () => {
+  const [productDetailData, setProductDetailData] = useState({});
+
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+  const settings2 = {
+    dots: false,
+    infinite: true,
+    arrow: true,
+    speed: 500,
+    slidesToShow: 4,
+    slidesToScroll: 1,
+  };
+
+  useEffect(() => {
+    fetch("http://localhost:3000/Data/DetailpageData.json")
```

Comment:
아래와 같이 포트번호 생략가능합니다! 
포트 번호가 달라지면 각 fetch 주소를 바꾸어야 하기때문에 로컬주소는 삭제해주시는게 편리합니다 :) 
```suggestion
    fetch("/Data/DetailpageData.json")
```

Reply:
아~! 넵! 알겠습니다! 말씀하신대로 수정했습니다!